### ISSUE TITLE
[FEA] Native windows 10 RAPIDS.AI package not dependent on WSL2

### ISSUE THREAD

**Issue by torstendahlen** [rapidsai/cudf#7592]
With constantly updating windows feature updates within the Insider Preview it is impossible to get RAPIDS.AI to install and run within the WSL2 subsystem. Also Nvidia driver updates does create problems aswell.

I believe that we need a rapids.ai package that is independet from using the WSL2 on windows but rather in native windows python support.

**Comment by kkraus14** [comment]
Closing as duplicate of #28
